subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -58,6 +58,33 @@ config BT_LLL_VENDOR_OPENISA
 	help
 	  Use OpenISA Lower Link Layer implementation.
 
+# First redefine the experimental symbols. 
+# We define them here due to avoid declaring them right below the non-experimental
+# definitions. In that case, both would show up, which is an issue in the Kconfig library.
```

Comment:
@tejlmand , as discussed it would be nice if we could add a reference to the KConfig issue here.

Reply:
yep, still trying to reproduce using a small snippet to fully understand the nature of the problem.

Reply:
See issue / explanation of this behavior here https://github.com/ulfalizer/Kconfiglib/issues/117